### DISCUSSION TITLE
Images can not be sent to MongoDB

### DISCUSSION THREAD

**Discussion by gwittm** [neuefische/questions#789] [Web] [answered]
Thank you for your help in #786!
We managed to send the images to cloudinary. But the image can´t be sent to mongo db. When logging the whole recipe, that includes also the key "imageUrl", the imageUrl is not even displayed at the log. Logging the imageUrl itself it returns undefined.
We appreciate any ideas!

https://github.com/gwittm/Digital-Recipe-Book/tree/Cloudinary-Gabi

**Comment by NikRoe** [comment, marked as answer]
Hi there,

thanks for creating the discussion and your patience. 

I was able to make it work on my end, but since I made several changes, it is best to work this out together.

I will come visit you shortly 👋
